version.cmake

```diff
@@ -1,4 +1,4 @@
 # Copyright (C) 2017 Prism Skylabs
 set(ConnectSDK_VERSION_MAJOR 1)
-set(ConnectSDK_VERSION_MINOR 12)
-set(ConnectSDK_VERSION_REVISION 2)
+set(ConnectSDK_VERSION_MINOR 13)
```

Comment:
Let's increment change version, in particular, increment minor in a separate pull-request just before delivering of the SDK. Otherwise we'll have delivered versions like 1.12, 1.20, 1.27 with gaps in minor